src/vcpkg/remove.cpp

```diff
@@ -259,7 +259,7 @@ namespace vcpkg::Remove
 
         if (has_non_user_requested_packages)
         {
-            msg::println_warning(msgAdditionalPackagesToRemove);
+            msg::println_warning(msgAdditionalPackagesToExport);
```

Comment:
That seems wrong; I think you meant to add the additional packages to export message rather than replace the packages to remove one.